platform/ios/src/MGLMapView_Private.h

```diff
@@ -15,6 +15,9 @@ extern const CGSize MGLAnnotationAccessibilityElementMinimumSize;
 
 - (mbgl::Map *)mbglMap;
 
+/** Converts NSTimeInterval to mbgl::Duration */
+mbgl::Duration MGLDurationInSeconds(NSTimeInterval duration);
+
```

Comment:
Not currently. We do need a good home for non-geometric utility functions. The macOS version of MGLMapView.mm also defines an `MGLDurationInSeconds()`. It’d be great if we could replace them with a single method somewhere in platform/darwin/src/. Considering that `NSTimeInterval` is defined in NSDate.h, how about adding a private NSDate+MGLAdditions.h that just contains this method for now but could in the future have a category on NSDate?